bentoml/cli/yatai_service.py

```diff
@@ -17,74 +17,88 @@
 
 import click
 
-
 logger = logging.getLogger(__name__)
 
 
 def add_yatai_service_sub_command(cli):
     # pylint: disable=unused-variable
 
-    @cli.command(help='Start BentoML YataiService for model management and deployment')
+    @cli.command(help="Start BentoML YataiService for model management and deployment")
     @click.option(
-        '--db-url',
+        "--db-url",
         type=click.STRING,
-        help='Database URL following RFC-1738, and usually can include username, '
-        'password, hostname, database name as well as optional keyword arguments '
-        'for additional configuration',
-        envvar='BENTOML_DB_URL',
+        help="Database URL following RFC-1738, and usually can include username, "
+        "password, hostname, database name as well as optional keyword arguments "
+        "for additional configuration",
+        envvar="BENTOML_DB_URL",
     )
     @click.option(
-        '--repo-base-url',
+        "--repo-base-url",
         type=click.STRING,
-        help='Base URL for storing BentoML saved bundle files, this can be a file '
+        help="Base URL for storing BentoML saved bundle files, this can be a file "
         'system path(POSIX/Windows), or a S3 URL, usually starting with "s3://"',
-        envvar='BENTOML_REPO_BASE_URL',
+        envvar="BENTOML_REPO_BASE_URL",
     )
     @click.option(
-        '--grpc-port',
+        "--grpc-port",
         type=click.INT,
         default=50051,
-        help='Port to run YataiService gRPC server',
-        envvar='BENTOML_GRPC_PORT',
+        help="Port to run YataiService gRPC server",
+        envvar="BENTOML_GRPC_PORT",
+    )
+    @click.option(
+        "--prometheus-port",
+        type=click.INT,
+        default=50052,
+        help="Port to run Prometheus server for metrics exporting",
+        envvar="BENTOML_PROMETHEUS_PORT",
```

Comment:
For our users' typical use-cases, they are deploying to Kubernetes cluster with prometheus server monitoring the `/metrics` endpoint.

I think this is a good attempt to provide more access to the users. However this might confuse users.  Let's remove this for now.